### ISSUE TITLE
Download issue

### ISSUE THREAD

**Issue by Jerin-Jacob98** [apna-college/Delta#149]
I am not able to download any files for the CSS project - Assets.
Link for the files : https://github.com/apna-college/Delta/tree/main/CSS/Project/assets

Please Help!

**Comment by rohit-hub07** [comment]
Download the files one by one. Click on the top right corner. Here is the screenshot
![Screenshot 2024-07-23 010905](https://github.com/user-attachments/assets/384f402c-a670-43a9-a0cc-fdd03837d185)

**Comment by Jerin-Jacob98** [comment]
Yes, I am aware of it :).  That button is not working for some reason 😅

**Comment by rohit-hub07** [comment]
Is the image is not loading or the button is not showing? Can you share a screenshot?
